services/blockchain-indexer/methods/dataService/transactions.js

```diff
@@ -18,7 +18,7 @@ const {
 	getTransactions,
 	getPendingTransactions,
 	postTransactions,
-	getCommandsParamsSchemas,
+	getSchemas,
```

Comment:
Would exposing this from some other file would be more appropriate as this will contain all type schemas including transactions?